src/app/plugins/garage.plugin.ts

```diff
@@ -0,0 +1,86 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType, IHttpResponse } from '../../common/types';
+
+const cheerio = require("cheerio")
```

Comment:
You should install the cheerio types located [here](https://www.npmjs.com/package/@types/cheerio) and utilize `import * as cheerio from 'cheerio';` instead.

Reply:
`import * as cheerio from 'cheerio'` is not recommended. https://github.com/cheeriojs/cheerio#loading

Reply:
This is assuming that the user is using javascript. We should remain consistent and use the latter import